code/modules/chemistry/tools/food_and_drink.dm

```diff
@@ -309,6 +309,38 @@
 	afterattack(obj/target, mob/user , flag)
 		return
 
+	get_desc(mob/user)
```

Comment:
I feel like you should be calling the parent via `. = ..()` as the first thing unless you have a good reason not to.

Reply:
Calling the parent creates an invisible <br> that i thought was annoying and only happens when the distance from the object is more than 2, and as far as I am aware food/snacks does not need the examine desc of reagents inside themselves. I could work around it but I couldnt see how snacks benefits from the parent proc's function